validate.go

```diff
@@ -439,14 +439,16 @@ func (v *Validator) HexColor(key, value string, message ...string) {
 func (v *Validator) Len(key, value string, min, max int, message ...string) {
 	msg := getMessage(message, "")
 
+	runeVal := []rune(value)
```

Comment:
Small thing: if you're creating an intermediate variable then you might as well use `length := len([]rune(value))`? That's only what it's used for.

There is also `utf8.RuneCountInString()`, which seems a bit faster:

	Original:
		BenchmarkLenASCII-4     300000000                4.04 ns/op
		BenchmarkLenMB-4        300000000                4.04 ns/op

	Original PR:
		BenchmarkLenASCII-4     30000000                48.1 ns/op
		BenchmarkLenMB-4        20000000                66.3 ns/op

	RuneCountInString:
		BenchmarkLenASCII-4     50000000                26.1 ns/op
		BenchmarkLenMB-4        50000000                32.7 ns/op

Not that it really matters all that much in this context, but don't see a reason not to use it?

Full patch with benchmark:

```
diff --git i/validate.go w/validate.go
index 64bb805..ec9846d 100644
--- i/validate.go
+++ w/validate.go
@@ -52,6 +52,7 @@ import (
 	"strconv"
 	"strings"
 	"time"
+	"unicode/utf8"
 
 	"github.com/teamwork/mailaddress"
 )
@@ -439,16 +440,16 @@ func (v *Validator) HexColor(key, value string, message ...string) {
 func (v *Validator) Len(key, value string, min, max int, message ...string) {
 	msg := getMessage(message, "")
 
-	runeVal := []rune(value)
+	l := utf8.RuneCountInString(value)
 
 	switch {
-	case len(runeVal) < min:
+	case l < min:
 		if msg != "" {
 			v.Append(key, msg)
 		} else {
 			v.Append(key, fmt.Sprintf(MessageLenLonger, min))
 		}
-	case max > 0 && len(runeVal) > max:
+	case max > 0 && l > max:
 		if msg != "" {
 			v.Append(key, msg)
 		} else {
diff --git i/validate_test.go w/validate_test.go
index 476c2be..1319d69 100644
--- i/validate_test.go
+++ w/validate_test.go
@@ -810,3 +810,17 @@ func TestErrorOrNil(t *testing.T) {
 		})
 	}
 }
+
+func BenchmarkLenASCII(b *testing.B) {
+	v := New()
+	for n := 0; n < b.N; n++ {
+		v.Len("x", "Hello, world, it's a sentence!", 0, 40)
+	}
+}
+
+func BenchmarkLenMB(b *testing.B) {
+	v := New()
+	for n := 0; n < b.N; n++ {
+		v.Len("x", "H€llo, world, it's a sentence¡", 0, 40)
+	}
+}
```